piety-promoted-v3/modernized/11-chapter-11.adoc

```diff
@@ -306,7 +306,7 @@ It seems as if I felt your resistance; and my captive spirit needs to be set fre
 Her sister replied, that she did all in her power to submit;
 did not dare to call in question the dispensation; was mercifully permitted to feel calm;
 and thought she could venture to say, '`I have resigned you.`'
-forever '`Ah,`' said Elizabeth, '`but have you made a cheerful surrender of me?
+'`Ah,`' said Elizabeth, '`but have you made a cheerful surrender of me?
```

Comment:
I looked in the original for this and didn't see "forever" in it....but I wasn't sure if should add it into the quote above "I have resigned you forever." I did what I thought seemed right! 😄

Reply:
I agree with leaving it out. 👍